dashboard/app/models/video.rb

```diff
@@ -20,6 +20,8 @@ class Video < ActiveRecord::Base
   include Seeded
 
   default_scope {order(:key)}
+  scope :english_locale, -> {where(locale: 'en-US')}
+  scope :current_locale, -> {where(locale: I18n.locale.to_s).or(Video.english_locale).unscope(:order).order("(case when locale = 'en-US' then 0 else 1 end) desc").limit(1)}
```

Comment:
is the `.limit(1)` necessary for this to work? If so, I'd much prefer to find a different way to do this that doesn't have that kind of hidden functionality.

However, if `find` works by grabbing the first ordered result, then I think this should work just fine.

Reply:
find_by just returns the first ordered result -- https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by

So I think you're right, the limit clause isn't necessary. I've removed it.